Source/OCMock/OCPartialMockObject.m

```diff
@@ -45,14 +45,19 @@ - (void)dealloc
 	[super dealloc];
 }
 
-- (NSString *)description
+- (NSObject *)realObject
 {
-	return [NSString stringWithFormat:@"OCPartialMockObject(%@)", NSStringFromClass(mockedClass)];
+	return realObject;
 }
 
-- (NSObject *)realObject
+- (NSString *)description
 {
-	return realObject;
+    NSString *name = [self OCMockName];
+    if (name)
+    {
+        return [NSString stringWithFormat:@"OCPartialMockObject(%@) - %@", [realObject class], name];
+    }
+    return [NSString stringWithFormat:@"OCPartialMockObject(%@)", [realObject class]];
```

Comment:
Not saying this is wrong, but I'd like to understand why you're replacing `NSStringFromClass(mockedClass)` with `[realObject class]`.

Reply:
That was an error. Fixed.